### ISSUE TITLE
Bug: Unexpected warnings for ignored license

### ISSUE THREAD

**Issue by fhg-isi** [FHPythonUtils/LicenseCheck#48]
### System info

- OS: Windows

### Describe the bug

I try to ignore the 'ZOPE PUBLIC LICENSE'

`licensecheck --ignore-licenses='ZOPE PUBLIC LICENSE'  `  

and would expect that no warnings are shown about it. However, I get many warnings:

### Expected outcome

Do not show 'License not identified ' warnings for licenses that are ignored. 

### Actual outcome

```
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE
```
Related: #46

**Comment by FredHappyface** [comment]
fixed in 2023.2

**Comment by fhg-isi** [comment]
Unfortunately, the issue is still present with version 2023.2

**Comment by FredHappyface** [comment]
Hmm I'll investigate further. Would you be able to provide me with more info please? Such as a link to your project or a requirements.txt?

Thank you 😊

**Comment by fhg-isi** [comment]
https://github.com/fraunhofer-isi/micat/blob/main/back_end/pyproject.toml

**Comment by FredHappyface** [comment]
Cheers for this, will be fixed in the next release :) 

![image](https://github.com/FHPythonUtils/LicenseCheck/assets/41634689/61fdf32c-b56b-47ff-a428-e248dd534737)

**Comment by FredHappyface** [comment]
Closed in 2023.3. Use as above :)

**Comment by fhg-isi** [comment]
![image](https://github.com/FHPythonUtils/LicenseCheck/assets/76424482/4b334fbb-7dac-42f4-b10a-0bdb39e46e51)
